src/components/pages/404-pages/404PagesPage.vue

```diff
@@ -1,55 +1,54 @@
 <template>
   <div class="not-found-pages row">
     <div class="col-md-12">
-      <div class="widget">
-        <div class="widget-header">
-          404 pages
-        </div>
-      </div>
       <div class="row">
         <vuestic-card
+          class="not-found-pages__cards"
           image="https://i.imgur.com/GzUR0Wz.png">
           {{ 'Advanced layout' }}
-          <p class="pt-3 mb-0">
+          <div class="not-found-pages__button-container  pt-3 mb-0">
             <router-link :to="{ path: '/pages/not-found-advanced' }">
               <button class="btn btn-primary not-found-pages__advanced-button">
                 {{ 'View Example' }}
               </button>
             </router-link>
-          </p>
+          </div>
         </vuestic-card>
         <vuestic-card
+          class="not-found-pages__cards"
           image="https://i.imgur.com/HttcXPi.png">
           {{ 'Simple' }}
-          <p class="pt-3 mb-0">
+          <div class="not-found-pages__button-container pt-3 mb-0">
             <router-link :to="{ path: '/pages/not-found-simple' }">
               <button class="btn btn-primary not-found-pages__advanced-button">
                 {{ 'View Example' }}
               </button>
             </router-link>
-          </p>
+          </div>
         </vuestic-card>
         <vuestic-card
+          class="not-found-pages__cards"
           image="https://i.imgur.com/dlcZMiG.png">
           {{ 'Custom image' }}
-          <p class="pt-3 mb-0">
+          <div class="not-found-pages__button-container pt-3 mb-0">
             <router-link :to="{ path: '/pages/not-found-custom' }">
               <button class="btn btn-primary not-found-pages__advanced-button">
                 {{ 'View Example' }}
               </button>
             </router-link>
-          </p>
+          </div>
         </vuestic-card>
         <vuestic-card
+          class="not-found-pages__cards"
```

Comment:
Can we store config in `data` and iterate using v-for?
```
data () {
  return [{
    imageUrl: 'https://i.imgur.com/dlcZMiG.png',
    label: 'Custom image',
    buttonTo: '/pages/not-found-custom',
    buttonLabel: 'View Example' <- maybe no need to put it in the config, seems to be the same
  }, { ... }]
}
```
This way you can reduce code boilerplate. No need to duplicate `not-found-pages__button-container pt-3 mb-0` and `btn btn-primary not-found-pages__advanced-button` for each item in the template.